charts/kong/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Improvements
 
 * Add custom label configuration option for Kong proxy `Ingress`.
+* Running `tpl` against user-supplied labels and annotations used in Deployment
```

Comment:
Do you mind extending this note to give examples of how this can be used? Might be more discoverable than the PR description for someone looking at the changelog.

Reply:
pushed